src/Middleware/ResponseCaching/src/ResponseCachingFeature.cs

```diff
@@ -5,10 +5,12 @@
 
 namespace Microsoft.AspNetCore.ResponseCaching
 {
+    /// <inheritdoc />
```

Comment:
```suggestion
    /// Default implementation for <see cref="IResponseCachingFeature" />
```

The methods are fine, but it's odd to have the same summary as the contract.